lib/main.coffee

```diff
@@ -11,6 +11,8 @@ Config = require './config'
 KernelManager = require './kernel-manager'
 Inspector = require './inspector'
 AutocompleteProvider = require './autocomplete-provider'
+zmq_services = require './zmq-services'
```

Comment:
Seems doable in retrospect, but I'll need to check after writing the gateway code.

`jupyter-js-services` provides a number of functions as globals. I'll need to make sure that any place those globals are accessed has a reasonable way to obtain a `ZMQGateway` instance from either main or the kernel manager.